app/src/main/java/fr/free/nrw/commons/customselector/ui/selector/ImageLoader.kt

```diff
@@ -138,6 +138,18 @@ class ImageLoader @Inject constructor(
                             is Result.TRUE -> {
                                 mapResult[imageSHA1] = Result.TRUE
                             }
+                            is Result.ERROR -> {
```

Comment:
@nicolas-raoul The app is running without any crashes, all test cases are also getting passed, just some addition of tests for these new lines would be better.